<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageRecognize</class>
 <widget class="QWidget" name="imageRecognize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="image_file">
       <property name="placeholderText">
        <string>输入图片地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="image_file_button">
       <property name="text">
        <string>图片地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startRecognize">
       <property name="text">
        <string>开始识别</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="image">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
